core/main/src/firebolt/firebolt_ws.rs

```diff
@@ -116,13 +134,42 @@ impl tungstenite::handshake::server::Callback for ConnectionCallback {
     > {
         info!("New firebolt connection {:?}", request.uri().query());
         let cfg = self.0;
+
```

Comment:
About 9 unsafe `unwrap()` in this function. Avoid using unsafe `unwrap()`

Reply:
Pre existing code and unwraps are mostly self created safe controlled objects.